pkg/common/utils/gang_utils.go

```diff
@@ -123,24 +125,53 @@ func GetTaskGroupsFromAnnotation(pod *v1.Pod) ([]v1alpha1.TaskGroup, error) {
 	return taskGroups, nil
 }
 
-func GetPlaceholderTimeoutParam(pod *v1.Pod) (int64, error) {
+func GetSchedulingPolicyParam(pod *v1.Pod) map[string]interface{} {
+	schedulingPolicyParams := make(map[string]interface{})
+	timeout := int64(0)
+	var timeoutErr = fmt.Errorf("")
+	var styleErr = fmt.Errorf("")
+	style := constants.SchedulingPolicyStyleParamDefault
 	param, ok := pod.Annotations[constants.AnnotationSchedulingPolicyParam]
 	if !ok {
-		return 0, nil
+		schedulingPolicyParams["placeholderTimeout"] = timeout
+		schedulingPolicyParams["placeholderTimeoutErr"] = timeoutErr
+		schedulingPolicyParams["schedulingStyle"] = style
+		schedulingPolicyParams["schedulingStyleErr"] = styleErr
```

Comment:
please do not add `placeholderTimeoutErr` and `schedulingStyleErr` into the return map, which makes the return value pretty odd. for easier access, we can return a struct such as:

```
type schedulingParameters struct {
  placeholderTimeout int64
  gangSchedulingStyle string
}
```